### DISCUSSION TITLE
View list of unbillable or billable Tasks

### DISCUSSION THREAD

**Discussion by adarshmadrecha** [madrecha/jamku#597] [QA] [answered]
How do I get list of all unbillable or billable tasks from Jamku?

**Comment by Divwork92** [comment, marked as answer]
There are 2 ways of viewing the list of billable and non-billable tasks 

## 1st - Task List
1. Go to Task List
2. Click on 'search option', from the drop down select 'billable status'
   ![image](https://user-images.githubusercontent.com/87230676/126489248-f507de68-d31c-4a7d-8fbe-a15c9d177ea5.png)
3. select billable or non billable accordingly after selecting billing status
   ![image](https://user-images.githubusercontent.com/87230676/126489617-c38fac70-f561-42e2-8d97-3c62467dd312.png)
4. click on `+ADD` and then click on refresh (cloud icon)

*You can see the list of tasks which are billable, and which are non-billable as per you selection.

## 2nd - Task Costing
1. Go to task costing page
2. select the date range and refresh
3. under 'is billable', click on the filer icon and select 0 and 1 
![image](https://user-images.githubusercontent.com/87230676/126490560-1e0bd683-4420-4ca3-8694-2251db95fbe3.png)

*0 - non billable tasks
*1 - billable tasks

You can even export in excel by placing your cursor on the cell, right click, select export
   ![image](https://user-images.githubusercontent.com/87230676/126490999-b1585438-7f57-4ecf-8d30-4c0568a5e1e3.png)

Users need Money Permission to use Task Costing. While Task list option does not require this permission.
If user has restricted "Report Filter" permission, then they will be able to view only tier only task or their team members tasks.
